src/Components/WeatherDetails/WeatherDetails.js

```diff
@@ -1,67 +1,107 @@
-import React from 'react';
+import React, { Component } from 'react';
 
+import Button from './WeatherShowMoreLessButton';
+import { FadeInTop } from '../../common/Animations/Animations';
```

Comment:
cant't it be resolved ??

Reply:
I know what you mean but I need to more practice with that library